app/models/register.rb

```diff
@@ -79,6 +79,37 @@ def show_category_link?
     register_phase == 'Beta' && category.present?
   end
 
+  def is_register_published_by_dcms?
+    Authority.where(id: authority_id).first&.government_organisation_key === 'D5'
+  end
+
+  def fields_readable
+    register_fields.map do |field|
+      # Make the field more readable.
```

Comment:
rather than assigning a new property to fields I would rather we have a function that applies the transformation to a single field, then call that where needed.

Reply:
I've spun these into their own functions in the `application_helper.rb` file so they can be called when needed.